db/migrate/20160619064544_change_memory_reserve_data_type_in_vm.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeMemoryReserveDataTypeInVm < ActiveRecord::Migration[5.0]
+  def up
+    change_column :vms, :memory_reserve, :bigint
```

Comment:
Just to clarify a little...I think that in the future anywhere we store "memory" or "disk" related columns, we should store bytes.  This includes memory/disk reserved/used/free/total/whatever.